src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4418,7 +4418,6 @@ private SymbolInfo GetNamedArgumentSymbolInfo(IdentifierNameSyntax identifierNam
                 return (object)tupleElement == null ? SymbolInfo.None : new SymbolInfo(tupleElement, ImmutableArray<ISymbol>.Empty, CandidateReason.None);
             }
 
-            // PROTOTYPE(list-patterns) SyntaxKind.ListPatternClause
```

Comment:
Why is this removed? Shouldn't we still be returning info for such syntax nodes?

Reply:
I don't think this applies anymore, because we won't be expecting a list pattern within a recursive one, so it has to be handled separately.

Reply:
Sure, but do we have a prototype comment covering needing to handle that?

Reply:
Julien has a draft out for this https://github.com/dotnet/roslyn/pull/53850